srcpkgs/godot/template

```diff
@@ -57,7 +57,8 @@ post_patch() {
 
 do_install() {
 	vlicense LICENSE.txt
-	vinstall ${FILESDIR}/godot.desktop 644 /usr/share/applications/
+
+	vinstall ${wrksrc}/misc/dist/linux/org.godotengine.Godot.desktop 644 /usr/share/applications/
```

Comment:
```suggestion
	vinstall misc/dist/linux/org.godotengine.Godot.desktop 644 /usr/share/applications/
```
wrksrc is PWD

Reply:
I followed your suggestion and removed ${wrksrc} from the vinstall command for the .desktop file, as it was redundant. I also applied the same change to the vinstall command for the .png icon to maintain consistency.